manifests/0000_90_kube-controller-manager-operator_04_servicemonitor-controller-manager.yaml

```diff
@@ -70,6 +70,26 @@ spec:
     tlsConfig:
       caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
       serverName: kube-controller-manager.openshift-kube-controller-manager.svc
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+      - action: drop
+        regex: etcd_(debugging|disk|request|server).*
+        sourceLabels:
+          - __name__
+      - action: drop
+        regex: rest_client_request_latency_seconds_(bucket|count|sum)
+        sourceLabels:
+          - __name__
+      - action: drop
+        regex: root_ca_cert_publisher_sync_duration_seconds_(bucket|count|sum)
+        sourceLabels:
+          - __name__
+    port: httpscpc
+    scheme: https
+    tlsConfig:
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
+      serverName: kube-controller-manager.openshift-kube-controller-manager.svc
```

Comment:
Please use the client-cert auth here as in https://github.com/openshift/cluster-kube-apiserver-operator/pull/1190

Reply:
thanks for letting me know - updated